config.defaults.yaml

```diff
@@ -51,3 +162,137 @@ docs:
       coordinates: [254.7213, 32.8341]
       period: 0.31087445597147706
       title: RR Lyrae c
+
+features:
```

Comment:
Since ra/dec are not "features" should they be separated as "metadata"?

Reply:
Well, we are using them as features (just as, e.g. info on the cross-matches), so they do belong here.

Reply:
But they are not used as a variable, right? I think we had discussed this in the past. Even if they belong here, should they be designtaed differently?